dispute.go

```diff
@@ -26,14 +26,42 @@ type DisputeParams struct {
 // DisputeEvidenceParams is the set of parameters that can be used when submitting
 // evidence for disputes.
 type DisputeEvidenceParams struct {
-	ProductDesc, CustomerName, CustomerEmail, CustomerIP, CustomerSig, BillingAddress, Receipt string
-	ShippingAddress, ShippingDate, ShippingTracking, ShippingDoc                               string
-	RefundPolicy, RefundPolicyDisclosure, RefundRefusalReason                                  string
-	CancellationPolicy, CancellationPolicyDisclsoure, CancellationRebuttal                     string
-	ActivityLog                                                                                string
-	ServiceDate, ServiceDoc                                                                    string
-	DuplicateCharge, DuplicateChargeReason, DuplicateChargeDoc                                 string
-	CustomerComm, UncategorizedText, UncategorizedFile                                         string
+	ProductDesc string
+
+	CustomerName  string
+	CustomerEmail string
+	CustomerIP    string
+	CustomerSig   string
+	CustomerComm  string
+
+	BillingAddress string
+
+	Receipt string
+
+	ShippingAddress  string
+	ShippingDate     string
+	ShippingTracking string
+	ShippingDoc      string
+
+	RefundPolicy           string
+	RefundPolicyDisclosure string
+	RefundRefusalReason    string
+
+	CancellationPolicy           string
+	CancellationPolicyDisclsoure string
```

Comment:
Also, oh man, did we really spell this "Disclsoure"? I'm going to blame this one on the fact that fields that are all on the same line are really hard to read ...